### PR TITLE
Garnett header behind a switch

### PR DESCRIPTION
**Pull request by zeftilldeath**
A few changes worth mentioning:

- Sections has become more (this will affect the current test)
- Two blank logos added
- The capitalisation will be strange as it's been forced with CSS - this will be resolved in a separate PR (https://github.com/guardian/frontend/pull/18611)
- AMP un-affected as far as I can see
- No JS works
- Lots of testing to be done
- Footer remains as is - this will be tackled separately

### CODE REVIEW

**Comment by NataliaLKB** [comment]
image

Oh my this is a big PR! Looking forward to this going out :)

File: static/src/stylesheets/layout/garnett-header/_menu-group.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        border-left: 1px solid $rule;
+    }
+}
+
+.menu-group--brand-extensions {
+    position: absolute;
+    right: $gs-gutter;
+    top: $gs-baseline;
+    bottom: 0;
+    width: 186px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/garnett-header/_menu-group.scss]: Anyway not to have this magic number? ✨

File: static/src/stylesheets/layout/garnett-header/_menu.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            top: $slim-nav-height;
+        }
+    }
+
+    .new-header--open & {
+        @include mq($until: desktop) {
+            @include menu-animation(0%);
+        }
+
+        // TODO: replace with sexy animation
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/garnett-header/_menu.scss]: Is this todo still important?

File: static/src/stylesheets/layout/garnett-header/_new-header.scss [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+        }
+    }
+
+    :hover,
+    :focus {
+        text-decoration-skip: ink;
+    }
+}
+
+.l-side-margins { // TODO remove this line
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/garnett-header/_new-header.scss]: Should we remove this line 😆

**Review by NataliaLKB — APPROVED** [review]
I have a few small questions or comments, but I think this is almost good to go out!

Awesome job @zeftilldeath

File: static/src/stylesheets/layout/garnett-header/_menu.scss (hunk 2 above)
Comment (zeftilldeath) [reply static/src/stylesheets/layout/garnett-header/_menu.scss]: If we decide to experiment with animation the TODO won't have been the reason : ) I will get rid.

File: static/src/stylesheets/layout/garnett-header/_new-header.scss (hunk 3 above)
Comment (zeftilldeath) [reply static/src/stylesheets/layout/garnett-header/_new-header.scss]: Yep!

File: static/src/stylesheets/layout/garnett-header/_menu-group.scss (hunk 1 above)
Comment (zeftilldeath) [reply static/src/stylesheets/layout/garnett-header/_menu-group.scss]: Unfortunately not, as it is all down to alignment with the logo. I can pop in a comment though.

**Pull request merged**